CONTRIBUTING.md

```diff
@@ -62,8 +65,9 @@ read it carefully.
 ## How Can I Talk to You?
 
 Discussion often happens in the issues and pull requests.
-In addition, there is a [Slack chat room](http://contact.fatiando.org) for the
-Fatiando a Terra project where you can ask questions.
+In addition, we also use Slack and other services to chat and organize project 
+activities.
+See https://www.fatiando.org/contact for links and more information.
```

Comment:
I just find the slack room link is missing on that page

Reply:
Or broken...

Reply:
https://github.com/fatiando/website/pull/45

Reply:
Thanks, Lu!